pkgs/applications/emulators/wine/sources.nix

```
@@ -160,16 +172,43 @@ rec {
     '';
   };
 
-  staging = fetchFromGitLab rec {
-    # https://gitlab.winehq.org/wine/wine-staging
-    inherit (unstable) version;
-    hash = "sha256-rXA/55rwQSJR247E4H7cQdTtXRmjomRbls7THV3jfcE=";
-    domain = "gitlab.winehq.org";
-    owner = "wine";
-    repo = "wine-staging";
-    rev = "v${version}";
+  yabridge = fetchurl rec {
+    # NOTE: Don't forget to change the hash for staging as well.
```

```suggestion
```

This line was mistakenly copy-pasted from `unstable`, or at least I really hope so.

---

No it wasn't a mistake, yabridge also pins a version of the staging patches that also needs the hash to be updated when the version is bumped.

---

```suggestion
    # NOTE: This is a pinned version with staging patches; don't forget to update them as well
```

… and actually I would be in favour of moving «staging» up in the attribute set, just under the  hash, because it needs to be noticed together with that comment.